docs/getting-started/devenv/using-visual-studio.md

```diff
@@ -1,13 +1,229 @@
+
+11. Edit `project.json` to add `"testRunner": "xunit",` after the `"frameworks"` section.
+
+12. Add a `LibraryTests.cs` class file to the **TestLibrary** project, add the `using` directives `using Xunit;` and `using Library;` to the top of the file, and add the following code to the class:
+    ```csharp
+    [Fact]
+    public void ThingGetsObjectValFromNumber() {
+        Assert.Equal(42, new Thing().Get(42));
+    }
+    ```
+    * Optionally, delete the `Program.cs` file from the **TestLibrary** project, and remove `"buildOptions": {"emitEntryPoint": true}` from `project.json`.
+
+   You should now be able to build the solution. 
+   
+13. On the **Test** menu, choose **Windows**, **Test Explorer**, and in Test Explorer choose **Run All**.
+   
+   The test should pass.
+
+### Writing the console app
+
+1. In Solution Explorer, open the context menu for the `src` folder, and add a new **Console Application (.NET Core)** project. Name it "App", and set the location to `Golden\src`.
+
+2. In the **App** project, open the context menu for the **References** node and choose **Add**,  **Reference**. 
+
+3. In the **Reference Manager** dialog, check **Library** under the **Projects**, **Solution** node, and then click **OK** 
+
+4. Open the context menu for the **References** node and choose **Restore Packages**.
+
+5. Open the context menu for the **App** node and choose **Set as StartUp Project**.
+
+6. Open the `Program.cs` file, add a `using Library;` directive to the top of the file, and then add `Console.WriteLine($"The answer is {new Thing().Get(42)}");` to the `Main` method.
+
+7. Set a breakpoint after the line that you just added.
+
+8. Press F5 to run the application..
+
+   The application should build without error, and should hit the breakpoint. You should also be able to check that the application output "The answer is 42.".
+
+A mixed .NET Core library and .NET Framework application
+--------------------------------------------------------
+
+Starting from the solution obtained with the previous script, execute the following steps:
+
+1. In Solution Explorer, open the `project.json` file for the **Library** project and replace `"frameworks": {
+    "netstandard1.5"` with `"frameworks": {
+    "netstandard1.4"`.
+
+2. In the **Library** project, open the context nmenu for the **References** node and choose **Restore Packages**.
+
+   The solution should still build and function exactly like it did before: the test should pass, and the console application should run and be debuggable.
+
+3. In Solution Explorer, open the context menu for the `src` folder, and choose **Add**. , **New Project**.
+
+4. In the **New Project** dialog, choose the **Visual C#** node, and then choose **Class Library**.
```

Comment:
I thought [4.5.2 corresponded to `netstandard1.2`](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms), so a dependency on a `netstandard1.4` library shouldn't work.

And it indeed seems to work for this simple example, but if I add `public FormattableString Foo() => $"";` to the library, and try to call that from a 4.5.2 Console Application, then it fails with a fairly cryptic message:

> CS7069 Reference to type 'FormattableString' claims it is defined in 'System.Runtime', but it could not be found